pkg/apis/v1/ec2nodeclass.go

```diff
@@ -61,7 +61,7 @@ type EC2NodeClassSpec struct {
 	// alias is specified, this field is required.
 	// NOTE: We ignore the AMIFamily for hashing here because we hash the AMIFamily dynamically by using the alias using
 	// the AMIFamily() helper function
-	// +kubebuilder:validation:Enum:={AL2,AL2023,Bottlerocket,Custom,Windows2019,Windows2022}
+	// +kubebuilder:validation:Enum:={AL2, AL2023, Bottlerocket, Custom, Windows2019, Windows2022}
```

Comment:
Are enums supposed to have spaces between them when configuring this with kubebuilder? From looking at the docs, I hadn't seen an example of this